### ISSUE TITLE
Opt out of private messages or disable them for specific groups

### ISSUE THREAD

**Issue by Athrunen** [WiIIiam278/HuskChat#150]
Would it be possible for players to opt out of receiving private messages or disallow players to send private messages to certain groups?

**Comment by WiIIiam278** [comment]
The aptly named /optoutmsg command lets a user stop receiving /r reply chains on their current group message. Is this what you're looking for?

**Comment by Athrunen** [comment]
No, I'm more interested in something more permanent. Essentials for example has /msgtoggle which opts you out of msg altogether. Or even within your own project you have the Channel permissions which limit who can send and receive what. I just find it weird that there is nothing to limit receiving private messages except turning off the entire feature.

Something like huskchat.msg.<group>.send to be able to send to a specific group might be interesting. As well as a toggle similar to essentials. And of course a bypass.
If any of the above is even feasible inside a chat plugin?

**Comment by WiIIiam278** [comment]
Setting huskchat.command.msg to false for a user will make them an invalid message target; they won't be able to send or receive messages, if that's what you're asking
